components/MashForms/ScreeningDecisionForm.tsx

```diff
@@ -52,8 +54,9 @@ const ScreeningDecisionForm = ({
       <h1>Make screening decision</h1>
       <Heading
         clientname={referral.clients.join(', ')}
-        timeleft="3 hours"
-        datetime={referral.createdAt}
+        filter={filter}
+        createdAt={referral.createdAt}
+        initialDecision={referral.initialDecision}
       />
```

Comment:
we can just pass the whole MashReferral object, let the component then decide how it wants to format clientName/s for example